micronutrientsupport/micronutrient-support-tool
# Pick and mix of tweaks

- close #716 
- update pull request default template to be simpler and (hopefully) shown by default this time...
- tooltip UI updates
- fixed mobile-view copyright being cropped

## To test
- viewing in mobile mode, open and use the ngx-feedback tool
- hover over a tooltip
- check that the copyright statement is not cropped in mobile mode
